contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Team Core Analytics
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- [Eric Duong (Team Lead, Full Stack Engineer)](/handbook/people/team/#eric-duong-software-engineer)
+- [Buddy Williams, Full Stack Engineer](/handbook/people/team/#buddy-williams-software-engineer)
+
+## Mission
+
+Make PostHog the best and most performant product analytics platform.
+
+## Responsibilities
+
+- Make sure data is correct and reliable
+- Make sure queries are quick
+- Increase the types of queries customers can do in PostHog (including frontend changes)
+
+## Customer
+
+- Product managers at scale-ups
```

Comment:
Probably engineers too? While right now we're focusing on a specific set of customers, we still have a solid base there and we likely want to continue with them?